pkgs/tools/security/gnupg/22.nix

```diff
@@ -36,6 +36,7 @@ stdenv.mkDerivation rec {
     ./tests-add-test-cases-for-import-without-uid.patch
     ./allow-import-of-previously-known-keys-even-without-UI.patch
     ./accept-subkeys-with-a-good-revocation-but-no-self-sig.patch
+    ./scdaemon_shared-access.patch # https://github.com/GPGTools/MacGPG2/blob/dev/patches/gnupg/scdaemon_shared-access.patch
```

Comment:
Please always add comments on new patches:

What does this patch do / why is it here, what's the upstream story for it (ideally linking to PRs, issues or discussions), and what is the condition of it being removed.

Reply:
Added description with links. Let me know if you would like to see something more.